NPM error when installing Covalent Core 1.0.0-beta.3-1

This repository's issues are reserved for feature requests and bug reports.

### Do you want to request a *feature* or report a *bug*?
BUG (Known issue)

#### Bug Report
Updating to the version 1.0.0-beta.3-1 produces an error in the console due to some dependencies of Flex Layout
```
npm ERR! peer dep missing: @angular/common@^2.3.5, required by @angular/flex-layout@2.0.0-beta.7
npm ERR! peer dep missing: @angular/core@^2.3.5, required by @angular/flex-layout@2.0.0-beta.7
npm ERR! peer dep missing: zone.js@^0.7.2, required by @angular/flex-layout@2.0.0-beta.7
```
This should be solved when @angular/flex-layout updates to Angular 4. See https://github.com/angular/flex-layout/issues/254

#### What is the expected behavior?
Installs without errors.

#### Which version of Angular and Material, and which browser and OS does this issue affect?

@angular/cli: 1.0.0
node: 6.9.5
os: darwin x64

```
  "dependencies": {
    "@angular/common": "4.0.2",
    "@angular/compiler": "4.0.2",
    "@angular/core": "4.0.2",
    "@angular/forms": "4.0.2",
    "@angular/http": "4.0.2",
    "@angular/material": "2.0.0-beta.3",
    "@angular/platform-browser": "4.0.2",
    "@angular/platform-browser-dynamic": "4.0.2",
    "@angular/router": "4.0.2",
    "@covalent/core": "1.0.0-beta.3-1",
    "@ngx-translate/core": "~6.0.0",
    "@ngx-translate/http-loader": "0.0.3",
    "hammerjs": "^2.0.8",
    "core-js": "^2.4.1",
    "rxjs": "^5.1.0",
    "zone.js": "^0.8.4"
  },
```

--- a/package.json
+++ b/package.json
@@ -59,7 +59,6 @@
     "@angular/common": "^4.1.0",
     "@angular/compiler": "^4.1.0",
     "@angular/core": "^4.1.0",
-    "@angular/flex-layout": "2.0.0-beta.7",
     "@angular/forms": "^4.1.0",
     "@angular/http": "^4.1.0",
     "@angular/material": "2.0.0-beta.3",

--- a/scripts/rollup.js
+++ b/scripts/rollup.js
@@ -31,7 +31,6 @@ gulp.task('rollup-code', '', function() {
     '@angular/platform-browser/animations': 'ng.platformBrowser.animations',
     '@angular/platform-browser-dynamic': 'ng.platformBrowserDynamic',
     '@angular/material': 'ng.material',
-    '@angular/flex-layout': 'ng.flexLayout',
 
     // Rxjs dependencies
     'rxjs/Subject': 'Rx',

--- a/src/platform/core/index.ts
+++ b/src/platform/core/index.ts
@@ -4,9 +4,6 @@ import { CommonModule } from '@angular/common';
 import { HttpModule, JsonpModule } from '@angular/http';
 import { FormsModule } from '@angular/forms';
 
-import { MaterialModule } from '@angular/material';
-import { FlexLayoutModule } from '@angular/flex-layout';
-
 /**
  * COMMON
  */

--- a/src/platform/core/package.json
+++ b/src/platform/core/package.json
@@ -38,7 +38,6 @@
     "Steven Ov <steven.ov@teradata.com>"
   ],
   "dependencies": {
-    "@angular/flex-layout": "^2.0.0-beta.7",
     "@angular/material": "2.0.0-beta.3"
   },
   "peerDependencies": {

